pkgs/top-level/php-packages.nix

```diff
@@ -370,7 +370,8 @@ lib.makeScope pkgs.newScope (self: with self; {
           configureFlags = [
             "--enable-dom"
           ];
-          patches = lib.optionals (lib.versionOlder php.version "8.2.14") [
+          # Add a PHP lower version bound constraint to avoid applying the patch on older PHP versions.
+          patches = lib.optionals (lib.versionOlder php.version "8.2.14" && lib.versionAtLeast php.version "8.1") [
```

Comment:
We generally do not want to care about versions not in NIxpkgs to make maintenance easier.

Reply:
Or at least that was my assumption in the past, in the end it is up to PHP maintainers. Maybe they are okay with that.

Reply:
I believe this is a compromise. When applying a patch, we should clearly know for which version (upper and lower bounds) it should be applied. In this particular place, I don't think this is an issue, but in projects like `fossar/nix-phps` aiming at maintaining old PHP versions alive, this is a huge problem.